### ISSUE TITLE
Improve accessibility of the feedback form

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#12634]
As per https://bugzilla.mozilla.org/show_bug.cgi?id=1864523, we need to update the feedback form to improve its accessibility. There are things that we cannot address, though, mainly around the radio buttons and how we group them ("1.  Keyboard navigation:" and "2. Grouping:"). That's because we want a single group of radio buttons.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDFRNT-295)

**Comment by ioanarusiczki** [comment]
@willdurand 

When navigating to the radio buttons when I choose to check one I press space. If I press on it again it won't un-check the radio button. 

I see below for the file report anonymously that I can do that. Or at the checkbox when the illegal content is chosen for reason.

**Comment by willdurand** [comment]
> When navigating to the radio buttons when I choose to check one I press space. If I press on it again it won't un-check the radio button.

That's expected.

**Comment by ioanarusiczki** [comment]
@willdurand 

Alright, then is there anything else I should be looking for? Keyboard nav looks good. About the grouping , visually nothing changed, at least not yet.

**Comment by willdurand** [comment]
I think that's enough for now. I'll ask the a11y team for another review.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDFRNT-295
